src/main/groovy/edu/wpi/first/gradlerio/frc/FRCPlugin.groovy

```diff
@@ -172,6 +186,17 @@ class FRCPlugin implements Plugin<Project> {
         }
     }
 
+    @CompileDynamic
```

Comment:
? This method can be compiled static

Reply:
No, it can't. `parseText` just returns an object, and when I look at the class its an internal class it uses.

Reply:
In groovy, you can use `[]` on any object, which returns null if it's not found. See: https://github.com/JacisNonsense/EmbeddedTools/blob/incubating/src/main/groovy/jaci/gradle/deploy/cache/Md5FileCacheMethod.groovy